### PR TITLE
function show lesson list

### PR DESCRIPTION
**Pull request by dovanhuy20071999**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by dovanhuy20071999** [comment]
ready

File: resources/views/pages/user/lesson_list.blade.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
+@extends('layouts.app')
+@section('content')
+<div class="container mt-5">
+    <div class="row">
+        <div class="col-md-12">
+            <div class="d-flex justify-content-between align-items-center activity">
+                <div><h1>LESSON LIST OF COURSE: {{strtoupper($course->name)}}</h1></div>
```
Comment (quoctd-0607) [inline resources/views/pages/user/lesson_list.blade.php]: i18n

File: resources/views/pages/user/lesson_list.blade.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                    @foreach ($lessons as $lesson)
+                    <li class="d-flex justify-content-between">
+                        <div class="d-flex flex-row align-items-center">
+                            <div class="ml-2">
+                                <a href="{{ route('lesson.detail', $lesson->id) }}"><h3 class="mb-0">{{ $lesson -> name}}</h3></a>
+                            </div>
+                        </div>
+                        <div class="d-flex flex-row align-items-center">
+                            <div class="d-flex flex-column mr-2">
+                                <span class="date-time">from course: {{$course->name}}</span>
```
Comment (quoctd-0607) [inline resources/views/pages/user/lesson_list.blade.php]: tương tự

**Comment by dovanhuy20071999** [comment]
ready

**Pull request merged**